README.md

```diff
@@ -0,0 +1,10 @@
+# HamWAN Infrastructure Configs
+
+## Developer Workstation Setup (Fedora)
+```
+sudo dnf -y install vagrant-libvirt rubygem-rexml @virtualization
+sudo systemctl enable --now libvirtd
+sudo usermod --append --groups libvirt `whoami`
+vagrant up --no-parallel
```

Comment:
Would be great to see CI for this.

Reply:
The actual tests will come in via Molecule, this is just a prereq for those tests to run.  It's my intention that the tests can run on developer workstations, before any commit or CI run.  Certainly no harm in having Molecule registered as a CI test too, aside from cost.